### ISSUE TITLE
Vagrant up wait on Bringing machine 'default' up with 'vmware_fusion' provider...

### ISSUE THREAD

**Issue by walkwizus** [hashicorp/vagrant#8580]
Please note that the Vagrant issue tracker is reserved for bug reports and
enhancements. For general usage questions, please use the Vagrant mailing list:
https://groups.google.com/forum/#!forum/vagrant-up. Thank you!

### Vagrant version
Vagrant 1.9.4

### Host operating system
MacOS Sierra 10.12.4

### Guest operating system
Debian Jessie 64

### Vagrantfile
```ruby
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|

  config.vm.box = "walkwizus/jessie64"
  config.vm.network "private_network", ip: "10.0.100.10"

  config.vm.synced_folder "www", "/var/www/html", owner: "www-data", group: "www-data"

end
```

### Debug output
https://gist.github.com/walkwizus/90f12a19702bf6aea043e4f890c98da9

### Expected behavior
I work with Vagrant and VMWare Fusion. When I try to run vagrant up, Vagrant wait on : wait on Bringing machine 'default' up with 'vmware_fusion' provider...

Thanks for you help

**Comment by chrisroberts** [comment]
Hi there. Are you able to properly start VMs using vmware directly?

**Comment by walkwizus** [comment]
Hi,
Yes, vmware works perfectly. I've downgraded Vagrant to 1.8.5 and everything work but I don't know what appened with the new version of Vagrant.

**Comment by briancain** [comment]
@walkwizus Does this behavior still occur if you upgrade vagrant and the vmware fusion provider to the latest version?

**Comment by chrisroberts** [comment]
Hey there,

I am going to close this due to lack of response. If this is still occurring, please open a [_new issue_](https://github.com/mitchellh/vagrant/issues/new) and follow the provided issue template that appears when you click the "New Issue" button. This will help us in getting a reproduction and fix. Thanks! 😄
